tasks/utils/envs/replaceTemplateVars.js

```diff
@@ -1,5 +1,16 @@
 const { execSync } = require('child_process')
-const { template } = require('@keg-hub/jsutils')
+const { template, get } = require('@keg-hub/jsutils')
+const { getTapConfig } = require('@keg-hub/cli-utils')
+const nodePath = require('path')
+
+/**
+ * @param {String} path - client app path
+ * @returns {String} - tap alias for tap at path
+ */
+const getAppAlias = path => get(
```

Comment:
Random comment, but I think this is awesome. I know there are people out there that would hate it. 
I'm pretty sure there was a time when you would have given me grief over it. Good to see you've come around